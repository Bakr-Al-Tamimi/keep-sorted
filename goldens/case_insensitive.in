Case-insensitive sorting:
// keep-sorted-test start case=no
'/bar',
'/quux',
'/foo',
'/Baz',
'/Quux/etc',
// keep-sorted-test end

Case-insensitive sorting with mixed-case duplicates:
// keep-sorted-test start case=no
Bar
baz
foo
bAz
bar
baZ
spam
// keep-sorted-test end

Case-insensitive sorting with numeric sorting:
// keep-sorted-test start case=no numeric=yes
bar_1
bAR_5
BAR_100
bar_5
// keep-sorted-test end
