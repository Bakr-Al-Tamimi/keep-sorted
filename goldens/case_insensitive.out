Case-insensitive sorting:
// keep-sorted-test start case=no
'/bar',
'/Baz',
'/foo',
'/quux',
'/Quux/etc',
// keep-sorted-test end

Case-insensitive sorting with mixed-case duplicates:
// keep-sorted-test start case=no
Bar
bar
bAz
baZ
baz
foo
spam
// keep-sorted-test end

Case-insensitive sorting with numeric sorting:
// keep-sorted-test start case=no numeric=yes
bar_1
bAR_5
bar_5
BAR_100
// keep-sorted-test end
